tests/test_resources/test_project.yml

```diff
@@ -124,4 +124,6 @@ deployment:
             - "VPN"
           production:
             - "Outside-World"
+        tls:
+          all: "le-other-prod-wildcard-cert"
```

Comment:
Do you indeed want to explicitly specify this everywhere, or introduce a default?

Reply:
Yeah this is just to test if the "other" tls shows up again. There's still the default:
https://github.com/Vandebron/mpyl/pull/276/files#diff-9c158608d7b7b263c5851249606562b09a229bfd0c36de35bb26748cca07303aR65-R67